monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,80 @@
+[id="configuring-the-monitoring-stack"]
+= Configuring the monitoring stack
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: configuring-the-monitoring-stack
+
+toc::[]
+
+This section explains what is supported for the monitoring of user-defined projects. It also shows how to configure the monitoring stack, and demonstrates several common configuration scenarios.
```

Comment:
For assembly introductions, you could change "section" to "document", since the reference is to the whole assembly.

Reply:
Done.